includes/batch.inc

```diff
@@ -117,9 +110,10 @@ class IslandoraPagedContentPDFBatchParentObject extends IslandoraScanBatchObject
   public function batchProcess() {
     module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
     $file = file_load($this->objectInfo['pdf_file']);
+    $this->label = $this->getTitle();
```

Comment:
Spoke to Brandon about this; `$this->getTitle()` does `$this->getMods()`, so makes more sense to make the MODS first. Dunno why we're doing it differently elsewhere. Still want to make sure `getMods()` is being called if a label is set, so still calling it explicitly as well.